calebwagner/uteach-ilearn-server
# Cjw creating fixtures

#### Changes Made

1. Added the following files to `uteachilearn/fixtures` directory:
> `appusers.json`
> `categories.json`
> `comments.json`
> `connections.json`
> `meetings.json`
> `posts.json`
> `saveposts.json`
> `tokens.json`
> `users.json`

1. Modified the following files to include correct model attributes in `uteachilearn/models` ...
> `appuser.py`
> `comment.py`
> `connection.py`
> `meeting.py` 
> `message.py` 
> `post.py`
   ​
#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-creating-fixtures
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
- [ ] for modifications to models, check to ensure app_user has been changed to user
- [ ] for added files, ensure each fixture has correct relationships according to the ERD

4. View code file.
   > Confirm file modifications are present as indicated above.
